pkg/auth/rbac.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	log = logger.NewLoggerField("pkg/auth")
+	log = logger.NewLoggerScope("pkg/auth")
```

Comment:
```suggestion
	log = logger.NewLoggerScope("auth")
```
`pkg` as prefix doens't seem to make sense?